### DISCUSSION TITLE
after I use cooler dump to extract data, what exactly does count mean in the generated text file?

### DISCUSSION THREAD

**Discussion by tyewaichung** [open2c/cooler#268] [Q&A] [answered]
Hello, after I use cooler dump to extract data, what exactly does count mean in the generated text file?
like this：count
chrom1 start1 end1 chrom2 start2 end2 count
chr21 5030000 5040000 chr21 43300000 43310000 1

**Comment by thomas-reimonn** [comment, marked as answer]
Hello! Count is the number of contacts between those two genomic regions. This value is not normalized or balanced.
